x-pack/plugins/security_solution/public/common/components/alert_count_by_status/alert_count_by_rule_by_status.tsx

```diff
@@ -45,6 +50,7 @@ type GetTableColumns = (
 ) => Array<EuiBasicTableColumn<AlertCountByRuleByStatusItem>>;
 
 const KIBANA_RULE_ALERT_FIELD = 'kibana.alert.rule.name';
+const KIBANA_WORKFLOW_STATUS = 'kibana.alert.workflow_status';
```

Comment:
this and the one above it area available in `body/renderers/constants.ts`, `SIGNAL_RULE_NAME_FIELD_NAME` and `SIGNAL_STATUS_FIELD_NAME`

Reply:
oh just seeing this. fixed!